webrender/src/platform/macos/font.rs

```diff
@@ -358,7 +360,31 @@ impl FontContext {
                 let glyph = key.index as CGGlyph;
                 let bitmap = is_bitmap_font(ct_font);
                 let (x_offset, y_offset) = if bitmap { (0.0, 0.0) } else { font.get_subpx_offset(key) };
-                let metrics = get_glyph_metrics(ct_font, None, glyph, x_offset, y_offset, 0.0);
+                let mut shape = FontTransform::identity();
+                if font.flags.contains(FontInstanceFlags::SYNTHETIC_ITALICS) {
+                    shape = shape.synthesize_italics(OBLIQUE_SKEW_FACTOR);
+                }
+                let transform = if shape.is_identity() {
+                    None
+                } else {
+                    Some(CGAffineTransform {
+                        a: shape.scale_x as f64,
+                        b: -shape.skew_y as f64,
+                        c: -shape.skew_x as f64,
+                        d: shape.scale_y as f64,
+                        tx: 0.0,
+                        ty: 0.0,
+                    })
+                };
+                let extra_strikes = font.get_extra_strikes(1.0);
```

Comment:
Was this just a pre-existing bug, or is it related to this change?

Reply:
A pre-existing bug/oversight.